### ISSUE TITLE
 Changing Language unlocks blocks

### ISSUE THREAD

**Issue by muggl73** [adaptlearning/adapt-contrib-languagePicker#69]
I found a possible issue with language-picker and trickle / step-locking in my projects. 
When I change language using language-picker all blocks are suddenly marked as "unlocked" and the trickle-button isn't rendered anymore.
This problem occures with FW version 5.20, adapt-contrib-trickle version 6.2.0 and adapt-contrib-languagePicker version 5.0.1. With FW version 5.2 everything seems to work fine though.

**Comment by oliverfoster** [comment]
[69-languagepicker-trickle-unlock.zip](https://github.com/adaptlearning/adapt-contrib-languagePicker/files/9045638/69-languagepicker-trickle-unlock.zip)
I can't replicate. The courses need to have the same content, the same ids, the same trackingIds and have config.json:`_languagePicker._restoreStateOnLanguageChange = true`.
Example attached.

**Comment by muggl73** [comment]
Thanks for the example. It's working perfectly. I've looked through almost every line of code to see if there are any differences with mine, but can see none - except that in my course I link to a certain contentObject via "startID" and skip the start-menu completely. I'm beginning to think that the whole problem is due to the fact, that I created the course for FW-version 5.2 and updated to 5.20 and updated all plugins in the process. Maybe there are incompatibilities with the old configs and the newer plugin-/framework-versions? But this almost certainly seems not to be a bug in the component itself, I guess :-(
